src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk;Microsoft.NET.Sdk.Razor">
```

Comment:
You shouldn't need both of these right? Or did we forget to include the net sdk 😢

Reply:
Yeah, I guess we missed covering this in our take-over from Web.Sdk thing. We'd have to effectively move this from line from https://github.com/aspnet/websdk/blob/master/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.props#L20 to Razor.Sdk